lang/am-ET.json

```diff
@@ -1,120 +1,120 @@
-  "label.today": "Today",
-  "label.tracking-code": "Tracking code",
-  "label.type": "Type",
-  "label.unknown": "Unknown",
-  "label.username": "Username",
-  "label.value": "Value",
-  "label.view-details": "View details",
-  "label.websites": "Websites",
-  "label.yesterday": "Yesterday",
-  "message.active-users": "{x} current {x, plural, one {visitor} other {visitors}}",
-  "message.confirm-delete": "Are you sure you want to delete {target}?",
-  "message.confirm-reset": "Are you sure you want to reset {target}'s statistics?",
-  "message.copied": "Copied!",
-  "message.delete-warning": "All associated data will be deleted as well.",
-  "message.edit-dashboard": "Edit dashboard",
-  "message.failure": "Something went wrong.",
-  "message.get-share-url": "Get share URL",
-  "message.get-tracking-code": "Get tracking code",
-  "message.go-to-settings": "Go to settings",
-  "message.incorrect-username-password": "Incorrect username/password.",
-  "message.log.visitor": "Visitor from {country} using {browser} on {os} {device}",
-  "message.new-version-available": "A new version of umami {version} is available!",
-  "message.no-data-available": "No data available.",
-  "message.no-websites-configured": "You don't have any websites configured.",
-  "message.page-not-found": "Page not found.",
-  "message.powered-by": "Powered by {name}",
-  "message.reset-warning": "All statistics for this website will be deleted, but your tracking code will remain intact.",
-  "message.save-success": "Saved successfully.",
-  "message.share-url": "This is the publicly shared URL for {target}.",
-  "message.toggle-charts": "Toggle charts",
-  "message.track-stats": "To track stats for {target}, place the following code in the {head} section of your website.",
-  "message.type-delete": "Type {delete} in the box below to confirm.",
-  "message.type-reset": "Type {reset} in the box below to confirm.",
-  "metrics.actions": "Actions",
-  "metrics.average-visit-time": "Average visit time",
-  "metrics.bounce-rate": "Bounce rate",
-  "metrics.browsers": "Browsers",
-  "metrics.countries": "Countries",
-  "metrics.device.desktop": "Desktop",
-  "metrics.device.laptop": "Laptop",
-  "metrics.device.mobile": "Mobile",
-  "metrics.device.tablet": "Tablet",
-  "metrics.devices": "Devices",
-  "metrics.events": "Events",
-  "metrics.filter.combined": "Combined",
-  "metrics.filter.raw": "Raw",
-  "metrics.languages": "Languages",
-  "metrics.operating-systems": "Operating systems",
-  "metrics.page-views": "Page views",
-  "metrics.pages": "Pages",
-  "metrics.query-parameters": "Query parameters",
-  "metrics.referrers": "Referrers",
-  "metrics.screens": "Screens",
-  "metrics.unique-visitors": "Unique visitors",
-  "metrics.views": "Views",
-  "metrics.visitors": "Visitors"
+  "label.accounts": "መለያዎች",
+  "label.add-account": "መለያ ጨምር",
+  "label.add-column": "አምድ ጨምር",
+  "label.add-filter": "ማጣሪያ ጨምር",
+  "label.add-website": "ድር ጣቢያ ጨምር",
```

Comment:
ድር ጣቢያ?

Reply:
I was thinking "ድር ጣቢያ" for "website" and "ድህረገፅ" for "webpage"